hedera-node/src/main/java/com/hedera/services/state/migration/MigrationRecordsManager.java

```diff
@@ -99,6 +116,8 @@ public void publishMigrationRecords(final Instant now) {
 		if (curNetworkCtx.consensusTimeOfLastHandledTxn() == null) {
 			final var implicitAutoRenewPeriod = FUNDING_ACCOUNT_EXPIRY - now.getEpochSecond();
 			STAKING_FUND_ACCOUNTS.forEach(num -> publishForStakingFund(num, implicitAutoRenewPeriod));
+			// publish the migration records for 0.26.0, for granting free auto-renewal
+			publishContractFreeAutoRenewalRecords(now);
```

Comment:
As noted, this should happen in the `!= null` branch.

Reply:
Fixed as discussed